### PR TITLE
Added Faker and data generation functions

### PR DESCRIPTION
**Pull request by Sensationull**
@kleekich Could you review this for me when you get a chance

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const db  = require('./index.js');
+
+
+const samplePosts = [
+  {
+    uniqId: 123,
+    symbol: 'aapl',
+    name: 'Apple',
+    marketPrice: 100,
+    created_at: "2017-05-03T03:50:00Z",
```
Comment (kleekich) [inline database/seed.js]: You may use single quotation for consistency

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ ]
+
+let output = [
+  {
+    id: 000123,
+    symbol: 'aapl',
+    marketPrice: 24,
+  },
+]
+console.log('------->', db);
```
Comment (kleekich) [inline database/seed.js]: You may get rid of console.log after testing
```suggestion
```

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const samplePurchase = function() {
+  db.Buy.create(samplePosts)
+    .then(() => db.db.disconnect());
```
Comment (kleekich) [inline database/seed.js]: You might want to choose one of db to a different name? It seems very confused if you use db twice.

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+};
+
+const sampleBPower = function() {
+  db.BPower.create(output)
+    .then(() => db.db.disconnect());
```
Comment (kleekich) [inline database/seed.js]: Like previous one, you might want to choose one of db to a different name.

File: database/seed.js [hunk 5]
```diff
@@ -18,6 +18,7 @@ let output = [
     marketPrice: 24,
   },
 ]
+
 console.log('------->', db);
```
Comment (kleekich) [inline database/seed.js]: You might want to get rid of console.log after testing

File: database/index.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 var mongoose = require('mongoose');
-const mongoUri = 'mongodb://localhost/buying';
+const mongoUri = 'mongodb://127.0.0.1:27017/buyingp';
 const db = mongoose.connect(mongoUri, { useNewUrlParser: true });
 
+console.log('this is db', db);
```
Comment (kleekich) [inline database/index.js]: You might want to delete this.

File: database/seed.js (hunk 3 above)
Comment (kleekich) [inline database/seed.js]: You might want to choose one of db to a different name? It seems very confused if you use db twice. Maybe first db to Database as sounds like a module?

**Review by kleekich — CHANGES_REQUESTED** [review]

File: database/seed.js (hunk 1 above)
Comment (kleekich) [inline database/seed.js]: You may use single quotation for consistency
```suggestion
    created_at: '2017-05-03T03:50:00Z',
```

**Review by kleekich — CHANGES_REQUESTED** [review]
I updated my comments with suggestions. Take a look and feel free to apply those changes. You definitely want to change to db though.

File: database/index.js (hunk 6 above)
Comment (Sensationull) [reply database/index.js]: Will do!

File: database/seed.js (hunk 3 above)
Comment (Sensationull) [reply database/seed.js]: This was an old comment but I did get it taken care of! Thanks

**Pull request merged**